templates/macros/helpers.html

```diff
@@ -12,6 +12,12 @@
   {{attr|xmlattr}}
 {%- endmacro -%}
 
+{%- macro format_paragraphs(description) -%}
+  {%- for paragraph in description -%}
+    <p>{{paragraph}}</p>
+  {%- endfor -%}
+{%- endmacro -%}
```

Comment:
As this is a generic macro, it may not be a description being passed to it.
```suggestion
{%- macro format_paragraphs(paragraphs) -%}
  {%- for paragraph in paragraphs -%}
    <p>{{paragraph}}</p>
  {%- endfor -%}
{%- endmacro -%}
```